index.html

```diff
@@ -30,8 +30,11 @@ <h1>Romcal</h1>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
     <script src="dist/romcal.bundle.min.js"></script>
     <script>
+      var currentYear = new Date().getFullYear();
+
       var dates = Romcal.calendarFor({
-        year: 2018
+        year: currentYear,
+          locale: 'fr'
```

Comment:
yep, in fact this was a test and I forgot to remove it. Good catch!

Reply:
I would suggest againt removing it altogether, but modify it to check the browser locale and use that. `romcal` defaults to `enUs` when that particular language is not available.

Reply:
It's a nice to have that will be added in a separated PR. We also need to check front-end side what locales are available on Romcal. In the meantime I will just remove it.

Reply:
And also because there might need a bit of work backend side: https://github.com/romcal/romcal/issues/26#issuecomment-558401426

Reply:
Just to point out, when I first 

> > @pejulian, I also notice that you didn't transpose the cache feature for dates, on the new es6 codebase. Is it intentional, or something that you didn't have time to include again?
> 
> As @pejulian is currently AWOL, do what you think is the best. 😃

The original intent was to convert the entire code base into Typescript. I must admit however that I only managed a partial conversion of the code due to my limited knowledge at the time. Since then, I have come a long way in Node module development with Typescript and have better ideas of how the code base may be transformed into a type safe structure.

Reply:
Thanks for your feedback @pejulian! Now that I got deeper into the codebase, I was also considering to switch it to Typescript. In the meantime I chose to introduce Flow (in my last PR #158), so we can continue to improve the codebase litle by litle, with the goal to switch then the entire things to Typescript someday.